### PR TITLE
Indicator performs wifi scan upon popup open

### PR DESCRIPTION
**Pull request by razcakappa**
I'm just checking the accuracy of the way it is implemented on my fork.

### CODE REVIEW

**Comment by razcakappa** [comment]
Need approval please

**Comment by Djaler** [comment]
This is fix #70

File: src/Indicator.vala [hunk 1]
```diff
@@ -83,7 +84,32 @@ public class Network.Indicator : Wingpanel.Indicator {
     }
 
     public override void opened () {
-        // TODO
+        if(nm_client == null){
```
Comment (donadigo) [inline src/Indicator.vala]: Space before `(`.

**Comment by razcakappa** [comment]
@donadigo It seems the issue is closed right? So my input is no longer required right?

**Pull request closed without merging**